packages/google_mlkit_text_recognition/android/build.gradle

```diff
@@ -41,8 +41,8 @@ android {
 
 dependencies {
     implementation 'com.google.mlkit:text-recognition:16.0.0'
-    compileOnly 'com.google.mlkit:text-recognition-chinese:16.0.0'
-    compileOnly 'com.google.mlkit:text-recognition-devanagari:16.0.0'
-    compileOnly 'com.google.mlkit:text-recognition-japanese:16.0.0'
-    compileOnly 'com.google.mlkit:text-recognition-korean:16.0.0'
+    implementation 'com.google.mlkit:text-recognition-chinese:16.0.0'
```

Comment:
why did you change it from compileOnly to implementation?

Reply:
It is because you need to include those in your `android/app/build.gradle` just as we have it in our example app: https://github.com/flutter-ml/google_ml_kit_flutter/blob/develop/packages/example/android/app/build.gradle#L57-L60

Also, that is mentioned in the README: https://github.com/flutter-ml/google_ml_kit_flutter/tree/develop/packages/google_mlkit_text_recognition#adding-language-package-dependencies

Discard those lines from this PR to get the other changes merged.